### ISSUE TITLE
Sponge menu on top left when using an iPad is almost unusable

### ISSUE THREAD

**Issue by dualspiral** [SpongePowered/sponge_docs_theme#13]
When using my iPad (and, I imagine, any tablet, but I don't have any other tablet to test with) to navigate the docs, tapping on the "Sponge" menu on the top left opens the menu, but also acts as a link to the docs home page, meaning that I can't choose an item from the dropdown menu. In order to be able to use the menu, I have to long press on the link, which seems counter intuitive.

It would be nice that if a tablet is being used, tapping the menu button doesn't also take you to the docs home page.

(As an aside, it also does this on the Sponge forums, but because that's all AJAX like, the menu stays open as you are redirected to the forums home page so is less of an issue. Unexpected, but less of an issue.)

**Comment by Tzky** [comment]
So either converting the dropdown to be ``onclick`` instead of ``hover`` or removing the link to ``docs.spongepowered.org`` on the logo would solve your issue?

**Comment by dualspiral** [comment]
Yes, but that would have the (possibly unintended) side effect of stopping
that link working for everyone else too - I don't know how critical that is
to you or others.
Instead, is it worth possibly looking into cancelling the link event using
the touch events (
https://developer.mozilla.org/en-US/docs/Web/API/Touch_events)?
